cgi-bin/ljlib.pl

```diff
@@ -242,30 +250,6 @@ sub gearman_client {
     return $client;
 }
 
-sub mogclient {
-    return $LJ::MogileFS if $LJ::MogileFS;
```

Comment:
We'll remove it and I'll update the ~28 other places. I just didn't yet, this diff isn't final.